mailbox/api/src/main/java/org/apache/james/mailbox/model/SearchQuery.java

```diff
@@ -633,6 +633,10 @@ public static Criterion flagIsUnSet(Flag flag) {
         return new FlagCriterion(flag, BooleanOperator.unset());
```

Comment:
If you had this method, I would modify flagIsSet and flagIsUnset to use this new static method flag.